packages/block-editor/src/components/block-canvas/index.js

```diff
@@ -36,29 +42,37 @@ export function ExperimentalBlockCanvas( {
 					ref={ contentRef }
 					className="editor-styles-wrapper"
 					tabIndex={ -1 }
-					style={ { height } }
+					style={ {
+						height: '100%',
+						width: '100%',
+					} }
 				>
 					{ children }
 				</WritingFlow>
-			</>
+			</BlockTools>
```

Comment:
Only downside here is that we'll have two divs (BlockTools div and WritingFlow div), we could potentially merge the two later.

Reply:
For non iframe, maybe, but I don't think it's that important. For iframe writing flow already does not produce a div.